src/calculateRank.js

```diff
@@ -36,6 +38,8 @@ function calculateRank({
     PRS_WEIGHT = 3;
   const ISSUES_MEDIAN = 25,
     ISSUES_WEIGHT = 1;
+  const REVIEWS_MEDIAN = 10,
```

Comment:
```suggestion
  const REVIEWS_MEDIAN = 2,
```

Reply:
I have updated reviews median value and tests. Feel free to merge.